app/src/main/java/tech/relaycorp/letro/utils/Logger.kt

```diff
@@ -0,0 +1,40 @@
+package tech.relaycorp.letro.utils
+
+import android.util.Log
+import tech.relaycorp.letro.BuildConfig
+import java.lang.Exception
+import javax.inject.Inject
+
+interface Logger {
+    fun e(tag: String, message: String, exception: Exception)
+    fun w(tag: String, exception: Exception)
+    fun d(tag: String, message: String)
+    fun i(tag: String, message: String)
+    fun println(message: String)
```

Comment:
How's this different from `Logger.i()`? I can see their implementations are different, but my question is mainly why this is needed.

Reply:
I use it only in testing. Doesn't matter, I can use Logger.i() instead